### ISSUE TITLE
FR: Change color of only a portion of text

### ISSUE THREAD

**Issue by Eusorph** [zsviczian/obsidian-excalidraw-plugin#1991]
**Is your feature request related to a problem? Please describe.**
I work with long text in Exalidraw and I need the ability to change just a portion of text and not the whole content of the frame. I know I can turn a text frame into a file and thus do what I want, but it would be faster and easier for me to work with just text frames. 
If, for example, Exalidraw worked with the Editing Toolbar plugin, it would be great.

**Describe the solution you'd like**
1. create a text frame
2. write several paragraphs
3. select part of a paragraph and change color (plus apply other format like strikethough would be great.)

**Describe alternatives you've considered**
Create text file and embed them

**Comment by zsviczian** [comment]
yes, would be nice, but opens a whole can of worms that I don't want to open. I am afraid, sooner or later the Excalidraw team will open this can of worms, and then I'll have no choice but to deal with it. The issue is... currently text is handled as plain text which sits naturally with Markdown and that fits well with Obsidian. If Excalidraw will implement some sort of richtext format for text coloring that will have all sorts of implications on how links are parsed and overall dissonance with Obsidian... so for now, I am very happy with not having multi color text. If I need to highlight a word, I find other ways of doing so (e.g. underline, highlighter, box over, or write over with another text element). I use Excalidraw extensively and in practice this does not seem a real big drawback to me, especially considering all the headache that would come would multicolor text be implemented....

**Comment by Eusorph** [comment]
I understand. What about Markdown? All I would I need right now is for the text to respond to the Commentator plug-in in Obsidian. It would make my life much, much easier. Any way to get that?

**Comment by zsviczian** [comment]
:) Markdown does not support colors, but yes, formatting with markdown would also be nice. sorry, but it is not currently in the cards.

**Comment by panknows** [comment]
Any updates?
